liquid-rust-typeck/src/checker.rs

```diff
@@ -302,6 +305,33 @@ impl<'a, 'tcx> Checker<'a, 'tcx> {
         Ok(())
     }
 
+    fn check_assert(
+        &mut self,
+        env: &mut TypeEnv<'tcx>,
+        cursor: &mut Cursor,
+        cond: &Operand,
+        expected: &bool,
```

Comment:
```suggestion
        expected: bool,
```
https://rust-lang.github.io/rust-clippy/master/index.html#trivially_copy_pass_by_ref